shell/scripts/build-pkg.sh

```diff
@@ -13,7 +14,15 @@ if [ ! -d ${SHELL_DIR} ]; then
 fi
 
 VERSION=$(cd pkg/$1; node -p -e "require('./package.json').version")
+
+if [[ $COMMIT_BRANCH == "master" ]]; then
+  VERSION="latest"
```

Comment:
Think this one should still contain the version from package.json and be called `head` (to support https://github.com/rancher/rancher/pull/38831)

```
VERSION="${VERSION}-head"
```

Reply:
Here is the plugin package provided for use with `v1/harvester/plugin-assets`，  

I will push a new [Branch](https://github.com/harvester/dashboard/tree/release-harvester-plugin-v1.0) to the Rancher Dashboard.   （This new branch will be used to build the v1.0.3 harvester plugin version embedded in rancher），  It's equivalent to pushing the` https://github.com/richard-cox/dashboard/tree/test-pluginize-rc-2 `branch to `https://github.com/rancher/dashboard/tree/release-harvester-plugin-v1.0` branch, but the version of `pkg/harvester/package.json` is v1.0.3 (The plugins embedded in rancher are built from this branch).


So I think it's fine to use` VERSION="latest"`, which is also consistent with the naming convention of `https://releases.rancher.com/dashboard/latest/index.html`

Reply:
Ok, think i understand.

In the new branch that supports the `1.0.3` plugin though we would need a way to provide a '-head` build which would align with https://github.com/rancher/rancher/pull/38831. I think we could do that via the package.json in that branch `<version>-head`, and then when ok to release it gets updated to `<version>`?

Reply:
Yes, I think so, 
In pre-release, package.json version = `1.0.3-head`, 
In release, package.json version=`1.0.3`